src/backend/aspen/workflows/nextstrain_run/run_nextstrain_mpx.sh

```diff
@@ -74,11 +74,11 @@ $aws s3 cp /mpox/config/build_czge.yaml "${s3_prefix}/build_czge.yaml"
 $aws s3 cp /mpox/config/include.txt "${s3_prefix}/include.txt"
 
 # run snakemake, if run fails export the logs from snakemake to s3
-(cd /mpox && snakemake --printshellcmds --configfile config/build_czge.yaml --resources=mem_mb=312320) || { $aws s3 cp /mpox/.snakemake/log/ "${s3_prefix}/logs/snakemake/" --recursive ; $aws s3 cp /mpox/logs/ "${s3_prefix}/logs/mpox/" --recursive ; }
+(cd /mpox && snakemake --printshellcmds --configfile config/build_czge.yaml --resources=mem_mb=312320) || { $aws s3 cp /mpox/.snakemake/log/ "${s3_prefix}/logs/snakemake/" --recursive ; $aws s3 cp /mpox/results/mpxv/filter.log "${s3_prefix}/logs/mpox/" --recursive ; }
```

Comment:
mpox logs is all over the place and this one seems most useful to have. I hope this works with the `--recursive`. I don't have a S3 bucket to test this out :(